SU-SWS/stanford_profile
# D8CORE-1201 Alert when leaving an unsaved node form

# READY FOR REVIEW

# Summary
- Alert to the user if they are about to leave the page after editing part of the page.

# Need Review By (Date)
- 2/12

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. clear caches
1. edit/create a new node without saving
1. before saving, attempt to change the page (click on another link or try to close the tab)
1. verify an alert appears to confirm.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
